dask/array/stats.py

```diff
@@ -0,0 +1,519 @@
+    if nan_policy != 'propagate':
+        raise NotImplementedError("`nan_policy` other than 'propagate' "
+                                  "have not been implemented.")
+
+    n = float(a.shape[axis])
+    b2 = kurtosis(a, axis, fisher=False)
+
+    E = 3.0 * (n - 1) / (n + 1)
+    varb2 = 24.0 * n * (n - 2) * (n - 3) / ((n + 1) * (n + 1.) * (n + 3) * (n + 5))  # [1]_ Eq. 1
+    x = (b2 - E) / np.sqrt(varb2)  # [1]_ Eq. 4
+    # [1]_ Eq. 2:
+    sqrtbeta1 = 6.0 * (n * n - 5 * n + 2) / ((n + 7) * (n + 9)) * np.sqrt((6.0 * (n + 3) * (n + 5)) /
+                                                                          (n * (n - 2) * (n - 3)))
+    # [1]_ Eq. 3:
+    A = 6.0 + 8.0 / sqrtbeta1 * (2.0 / sqrtbeta1 + np.sqrt(1 + 4.0 / (sqrtbeta1**2)))
+    term1 = 1 - 2 / (9.0 * A)
+    denom = 1 + x * np.sqrt(2 / (A - 4.0))
+    denom = np.where(denom < 0, 99, denom)
+    term2 = np.where(denom < 0, term1, np.power((1 - 2.0 / A) / denom, 1 / 3.0))
+    Z = (term1 - term2) / np.sqrt(2 / (9.0 * A))  # [1]_ Eq. 5
+    Z = np.where(denom == 99, 0, Z)
+    if Z.ndim == 0:
+        Z = Z[()]
+
+    # zprob uses upper tail, so Z needs to be positive
+    return delayed(KurtosistestResult, nout=2)(Z, 2 * distributions.norm.sf(np.abs(Z)))
+
+
+@doc_wraps(scipy.stats.normaltest)
+def normaltest(a, axis=0, nan_policy='propagate'):
+    if nan_policy != 'propagate':
+        raise NotImplementedError("`nan_policy` other than 'propagate' "
+                                  "have not been implemented.")
+
+    s, _ = skewtest(a, axis)
+    k, _ = kurtosistest(a, axis)
+    k2 = s * s + k * k
+    return delayed(NormaltestResult, nout=2)(k2, delayed(distributions.chi2.sf)(k2, 2))
+
+
+@doc_wraps(scipy.stats.f_oneway)
+def f_oneway(*args):
+    # args = [np.asarray(arg, dtype=float) for arg in args]
+    # ANOVA on N groups, each in its own array
+    num_groups = len(args)
+    alldata = da.concatenate(args)
+    bign = len(alldata)
+
+    # Determine the mean of the data, and subtract that from all inputs to a
+    # variance (via sum_of_sq / sq_of_sum) calculation.  Variance is invariance
+    # to a shift in location, and centering all data around zero vastly
+    # improves numerical stability.
+    offset = alldata.mean()
+    alldata -= offset
+
+    sstot = _sum_of_squares(alldata) - (_square_of_sums(alldata) / float(bign))
+    ssbn = 0
+    for a in args:
+        ssbn += _square_of_sums(a - offset) / float(len(a))
+
+    # Naming: variables ending in bn/b are for "between treatments", wn/w are
+    # for "within treatments"
+    ssbn -= (_square_of_sums(alldata) / float(bign))
+    sswn = sstot - ssbn
+    dfbn = num_groups - 1
+    dfwn = bign - num_groups
+    msb = ssbn / float(dfbn)
+    msw = sswn / float(dfwn)
+    f = msb / msw
+
+    prob = _fdtrc(dfbn, dfwn, f)   # equivalent to stats.f.sf
+
+    return delayed(F_onewayResult, nout=2)(f, prob)
+
+
+@doc_wraps(scipy.stats.moment)
+def moment(a, moment=1, axis=0, nan_policy='propagate'):
```

Comment:
There might be some interface differences, but a stable parallel general moments algorithm is defined here: https://github.com/dask/dask/blob/master/dask/array/core.py#L1542.  This could definitely be updated to support skipping nans if desired, which may be the only difference here.